lib/web_ui/lib/src/engine/platform_views/content_manager.dart

```diff
@@ -128,8 +128,9 @@ class PlatformViewManager {
       }
 
       _ensureContentCorrectlySized(content, viewType);
+      wrapper.append(content);
 
-      return wrapper..append(content);
+      return wrapper;
```

Comment:
revert

Reply:
Btw this was purposeful. Breaking this one-liner up was suggested by @yjbanov as a little more readable